openfe_benchmarks/utils.py

```diff
@@ -74,7 +74,69 @@ def generate_relative_network_from_names(ligands: Iterable[SmallMoleculeComponen
 
     return Network(edges)
 
+class RHFEBenchmarkSystem:
```

Comment:
We can probably make this a subclass of the other, but for now it really doesn't matter.